starknet-foundry/crates/cast/src/starknet_commands/multicall/run.rs

```diff
@@ -104,5 +138,5 @@ pub async fn run(
         }
     }
 
-    Ok(())
+    execute_calls(account, parsed_calls, max_fee).await
```

Comment:
Is it possible to await when calling `run`? I think it would make more sense

Reply:
Do you mean returning future here?

Reply:
yup. if you think it would make sense.

Reply:
In the next PR I've unified all commands to return `Result<Vec<FieldElement>>` and I would like to keep it that way